src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { z } from 'zod';
-import { RedteamAssertionTypes, redteamConfigSchema } from './redteam/types';
+import { RedteamAssertionTypes, RedteamConfig } from './redteam/types';
```

Comment:
This change was necessary to avoid circular dependency, now that `redteam.provider` supports `ProviderSchema`